sdk/storage/azure-storage-blob/azure/storage/blob/_serialize.py

```diff
@@ -176,50 +168,41 @@ def serialize_blob_tags_header(tags=None):
     return ''.join(components)
 
 
-def serialize_blob_tags(tags=None):
-    # type: (Optional[Dict[str, str]]) -> Union[BlobTags, None]
+def serialize_blob_tags(tags: Optional[Dict[str, str]] = None) -> BlobTags:
     tag_list = []
     if tags:
         tag_list = [BlobTag(key=k, value=v) for k, v in tags.items()]
     return BlobTags(blob_tag_set=tag_list)
 
 
-def serialize_query_format(formater):
+def serialize_query_format(formater: Union[str, DelimitedJsonDialect]) -> Optional[QuerySerialization]:
     if formater == "ParquetDialect":
-        qq_format = QueryFormat(
-            type=QueryFormatType.PARQUET,
-            parquet_text_configuration=' '
-        )
+        qq_format = QueryFormat(type=QueryFormatType.PARQUET, parquet_text_configuration=' ')  #type: ignore [arg-type]
```

Comment:
azure\storage\blob\_serialize.py:179: error: Argument "parquet_text_configuration" to "QueryFormat" has incompatible type "str"; expected "Optional[MutableMapping[str, Any]]"  [arg-type]

Reply:
Try an empty dictionary which might be approximately equivalent to `' '`

Reply:
So when passing in `{}`:
![image](https://github.com/Azure/azure-sdk-for-python/assets/101599632/f3c7be0c-13f2-4af7-81a5-9075d8c36ffd)
We can see in the diff that the `RequestBody` is only different because it includes the information about what the parquet text configuration was.

Passing in `None`:
![image](https://github.com/Azure/azure-sdk-for-python/assets/101599632/96f07900-b763-4221-9f51-3e9ce3a056eb)
So here it compeltely omits `ParquetTextConfiguration` from the body, but again this test has an assert against "expected data" and the query always gets back correct results.

So I think our options are:

1. Leave the ignore and run as-is
2. Change to `None` and re-record
3. Change to `{}` and re-record

But again I don't think we know strictly what the behavior of this is supposed to be like but it always gets the right data back 🤔

Reply:
Option 1.